docs/framework/wcf/feature-details/transaction-protocols.md

```diff
@@ -282,8 +282,8 @@ Windows Communication Foundation (WCF) implements WS-Atomic Transaction and WS-C
 <wst:RequestSecurityTokenResponse   
 xmlns:wsse="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-wssecurity-secext-1.0.xsd"   
 xmlns:wssu="http://docs.oasis-open.org/wss/2004/01/oasis-200401-wss-     wssecurity-utility-1.0.xsd"   
```

Comment:
The namespace definition here is also wrong. There's whitespace in the middle of the uri